CHANGES.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+# 0.8.0
+
+Add support for `--no-use-platform-suffix` as a complement to `--use-platform-suffix`. Without
+specifying either, auto-disambiguation is still used to add a platform suffix whenever the
+set of target platforms is not just the current platform, but you can also now force a platform
+suffix to never be used by specifying `--no-use-platform-suffix`.
```

Comment:
Pex needs this for the `--scie-name-style platform-parent-dir` option it added in https://github.com/pex-tool/pex/pull/2523 to work right.